packager/react-native-xcode.sh

```diff
@@ -74,7 +74,7 @@ DEST=$CONFIGURATION_BUILD_DIR/$UNLOCALIZED_RESOURCES_FOLDER_PATH
 if [[ "$CONFIGURATION" = "Debug" && "$PLATFORM_NAME" != "iphonesimulator" ]]; then
   PLISTBUDDY='/usr/libexec/PlistBuddy'
   PLIST=$TARGET_BUILD_DIR/$INFOPLIST_PATH
-  IP=$(ipconfig getifaddr en0)
+  IP=$(ipconfig getifaddr en0) || IP=$(ipconfig getifaddr en1)
```

Comment:
You could probably make this `IP=$(ipconfig getifaddr en0 || ipconfig getifaddr en1)`